epics-release.py

```diff
@@ -178,6 +178,8 @@ def ValidateArgs( repo, packageSpec, opt ):
                             keeptmp		= False	)
     parser.add_option(	"-r", "-R", "--release", dest="release",
                         help="release version string, ex. -r R1.2.3-0.1.0" )
+    parser.add_option(	"-u", "--url", dest="repo_url",
+                        help="repo URL, ex. -u git@github.com:/slac-epics/ADCore.git" )
```

Comment:
typo in example usage:
```suggestion
                        help="repo URL, ex. -u git@github.com:slac-epics/ADCore.git" )
```